mep/people/templates/people/member_list.html

```diff
@@ -20,6 +20,23 @@
         <label class="query">
             {% render_field form.query aria-label=form.query.label %}
         </label>
+        <fieldset>
+                <legend>Filter by:</legend>
+                <div class="toggle-filters">
+                    {% render_field form.has_card aria-label=form.has_card.label class+="sr-only sr-only-focusable" %}
```

Comment:
can't tell if `class+="..."` is some syntax I've never seen or an actual typo 😬  it does appear to render the classes regardless though?

Reply:
It's a really slick feature that lets you append classes (which seemed safer since I don't care about other things we add, just that those two are appended)